### PR TITLE
updating the iOS SDK to 1.4

### PR DESCRIPTION
**Pull request by mwestergaardsc**
updated the sdk version in podfile and updated the readme

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -5,6 +5,6 @@ target 'BitmojiSampleApp' do
     workspace 'BitmojiSampleApp.xcworkspace'
     project 'BitmojiSampleApp.xcodeproj'
     pod 'PINCache', '2.3'
-    pod 'SnapSDK', '1.3.2', :subspecs => ['SCSDKLoginKit', 'SCSDKBitmojiKit']
+    pod 'SnapSDK', '> 1.4', :subspecs => ['SCSDKLoginKit', 'SCSDKBitmojiKit']
```
Comment (davidxiasc) [inline ios/Podfile]: Let's make this a specific version

Comment (mwestergaardsc) [reply ios/Podfile]: okay, should i do the same for the other sample apps?

Comment (davidxiasc) [reply ios/Podfile]: yes please :)

**Review by Deloww — COMMENTED** [review]
Update Snap

File: ios/Podfile (hunk 1 above)
Comment (Deloww) [reply ios/Podfile]: Waleed []()

**Review by Deloww — COMMENTED** [review]
Update

**Review by Deloww — COMMENTED** [review]
[ios/README.md](ios/Podfile)

**Pull request merged**